src/com/lundekhan/stokk/StokkRoute.kt

```diff
@@ -0,0 +1,96 @@
+package com.lundekhan
+
+import com.lundekhan.gui.HtmlTemplates.respondHtmlShell
+import com.lundekhan.stokk.StokkHelper
+import io.ktor.application.call
+import io.ktor.request.receiveParameters
+import io.ktor.routing.Route
+import io.ktor.routing.get
+import io.ktor.routing.post
+import io.ktor.routing.route
+import kotlinx.html.*
+import kotlin.math.roundToInt
+
+fun Route.stokkRoute(): Route = route("/stokk") {
+
+    fun SECTION.stokkForm(
+        year: String = "",
+        startAmount: String = "",
+        interest: String = "",
+        monthlyDeposit: String = ""
+    ): Unit = form(method = FormMethod.post) {
+        header {
+            style = "padding:0;"
+            h3 { +"compoundcalc." }
+            small { +"Calculate the compound interest!" }
+        }
+        section {
+            numberInput {
+                name = "year"
+                placeholder = "year"
+                if (year.isNotEmpty()) value = year
+            }
+            section {
+                numberInput {
+                    name = "startAmount"
+                    placeholder = "startAmount"
+                    if (startAmount.isNotEmpty()) value = startAmount
+                }
+            }
+            section {
```

Comment:
Maybe we should create a extensionfunction to make those sections.

Reply:
I changed how the sections is done, please have a look